### ISSUE TITLE
Drop down main menu not working

### ISSUE THREAD

**Issue by drinkharder** [getgrav/grav-theme-pinpress#8]
I had dropdown to true, but when I click on BLOG at menu.  It's not taking me to my blog.  See it at http://drinkharder.com/grav/v12

**Comment by l3d00m** [comment]
You cannot click the first menu item when dropdown is enabled, this is not a bug I think.

**Comment by drinkharder** [comment]
Yeah, I've reported this to the theme creator so they can fix it.  In the
mean time I switch to other them as you can see www.drinkharder.com

**Comment by l3d00m** [comment]
I am aware, but why should they fix it if it isn't a bug?

I think you misunderstood me.

**Comment by drinkharder** [comment]
Oh.. Sorry ...., check www.drinkharder.com  and I'm pretty sure that you
can click on first menu when dropdown enabled.  I've tried many other
themes and they all allowed.

**Comment by l3d00m** [comment]
I prefer it the way it is currently tbh.

Looking in the code though it looks like they coded it the way you described it. [here](https://github.com/getgrav/grav-theme-pinpress/blob/develop/templates/partials/navigation.html.twig#L5) for anybody who's interested. If you remove the `data-toggle="dropdown"` from this line, it'll work as you described but it's not longer possible to use the menu on mobile.

The menu in this theme is it's biggest disadvantage, I have my troubles with it as well (see #7). Otherwise it's a decent theme.
